.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
+debug.svg
```

Comment:
Fwiw, this is still equivalent to `**/debug.svg`, as it doesn't have the leading `/`.
From the [`.gitignore` docs](https://git-scm.com/docs/gitignore#_pattern_format):

>For example, "`**/foo`" matches file or directory "`foo`" anywhere, the same as pattern "`foo`".